js/templates/reviews/review.html

```diff
@@ -0,0 +1,73 @@
+<% if (ob.error) { %>
+  <p class="clrTErr txCtr"><i class="ion-alert-circled"></i> <%= ob.polyT('listingDetail.errors.loadReview', { id: ob.ratingId, error: ob.error }) %></p>
```

Comment:
If the review fails to load, and error will be shown in it's place. This fixes various situations where the count would be off, since the user can see that there should be X reviews (ie: we don't want to show 4 for the number of reviews but only 3 load because 1 failed silently).